static/app/views/starfish/queries/useSpanMetricsSeries.tsx

```diff
@@ -23,19 +22,19 @@ export type SpanMetrics = {
 };
 
 export const useSpanMetricsSeries = (
-  span: Pick<IndexedSpan, 'group'>,
+  group: string,
   queryFilters: SpanSummaryQueryFilters,
   yAxis: string[] = [],
   referrer = 'span-metrics-series'
 ) => {
   const pageFilters = usePageFilters();
 
-  const eventView = span
-    ? getEventView(span, pageFilters.selection, yAxis, queryFilters)
+  const eventView = group
```

Comment:
m: How can group be `undefined` here if it's defined as `string` in `useSpanMetricsSeries`? We can remove the `Boolean(group)` cast too then.

Reply:
True, but in Step 2 I'm going to move the `group` parameter into `queryFilters`, since it's just a filter like anything else, so this code isn't long for this world